src/makeRobot.js

```diff
@@ -38,7 +38,54 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    version,
+    wheels,
+    'coords': {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    get info() {
+      return 'name: ' + this.name
+      + ', chip version: ' + this.version + ', wheels: ' + this.wheels;
+    },
+    goRight(x = 1) {
+      if (x >= 0) {
+        this.coords.x += x;
+      }
+
+      return this;
+    },
+    goLeft(x = 1) {
```

Comment:
```suggestion
    },

    goRight(x = 1) {
      if (x >= 0) {
        this.coords.x += x;
      }

      return this;
    },

    goLeft(x = 1) {
```

Add blank lines between methods, fix all such cases